win/rl/helper.cc

```diff
@@ -151,6 +157,10 @@ PYBIND11_MODULE(helper, m)
         ;
 
     py::class_<class_sym>(m, "class_sym")
+        .def_static(
+            "from_mlet",
+            [](char let) -> const class_sym * { return &def_monsyms[let]; },
+            py::return_value_policy::reference)
```

Comment:
This is... interesting. Was this to bypass const?

Reply:
NVM, I saw the comment in the commit.

Reply:
This is a bit of an unusual situation for pybind: NetHack maintains an array of class_sym structs. We invent a Python class for class_syms, but we don't actually want to create (or modify) any class_sym objects, just expose the entries of the array. Returning a pointer and using py::return_value_policy::reference makes pybind not take ownership.

What doesn't work is turning this into __init__ with the py::nodelete trick as in permonst (same situation otherwise). So we'll just use this static method.